particula/aerosol_dynamics/parcel.py

```diff
@@ -48,43 +38,68 @@ def name(self) -> str:
     def add_particle(self, particle_object):
         """Adds a particle to the parcel.
         Parameters:
-            particle_data (list): The radius of the particle.
+            particle_object (object)
         """
-        self._particle_data.append(particle_object)
+        self._particle_data = np.append(self._particle_data, particle_object)
 
     def create_and_add_particle(
-        self, name: str, radius, density=1e3, charge=0
+        self, name: str, radius, density=1e3 * u.kg / u.m**3,
+        charge=0 * u.dimensionless
     ):
         """creats and then Adds a particle to the parcel.
```

Comment:
Typos, and please add space between the top line and the parameters.

Reply:
done